cms/templates/studio_xblock_wrapper.html

```diff
@@ -59,7 +59,12 @@
     % if not show_preview:
         is-collapsed
     % endif
-    ">
+    "
```

Comment:
Locate waffle flag data in a place that does not require an xblock to exist on the page, so that we can check if they are enabled on creation